python/GafferUI/ShufflePlugValueWidget.py

```diff
@@ -125,7 +125,7 @@ def _updateFromPlug( self ) :
 	"description",
 	"""
 	The name(s) of the destination data to be created. Use \"${source}\" to insert the name of the source data.\n
-	For example, to prefix all data with \"user:\" set the destination to \"user:${source}\").
+	For example, to prefix all data with \"render:\" set the destination to \"render:${source}\").
```

Comment:
Again, I'm not sure `user:` _or_ `render:` make sense in this context - might be better to use something completely generic.